### ISSUE TITLE
Have a rootkit or similar that's taking over machine

### ISSUE THREAD

**Issue by coldcore321** [dragokas/hijackthis#145]
I don't know what logs are required for you to make a determination of infected for this computer. 
All I know is that  it's undetected by anti viruses, it's changing anything it wants-account passwords, registry, group policies, blocking me from protecting my ram, using processes like mad, greyed out many services so I can't access the entry, stops me from dling certain things like network monitors. It seems to come in on NAT. Is using MS edge to further itself(?), can see what I'm doing and will temporarily keep from doing whatever action. Now is attacking the email SNMP and POP's  on network. Bunches of unknown devices hooking up to network, hidden volumes on computer, taken over my phone no matter how much I reset it.  Etc..etc. etc,.

Please tell me what log files from where do you need to see from this computer to fix. I'll start with HJT log. 
[HOT POTATO.txt](https://github.com/dragokas/hijackthis/files/6261147/HOT.POTATO.txt)

**Comment by Sandor-Helper** [comment]
Hi,
If you need our assistance:
 
* Read carefully: [How to make a request for help in the PC cure section](https://github.com/dragokas/hijackthis/wiki/How-to-make-a-request-for-help-in-the-PC-cure-section%3F)
* Attach 'Collection-[Date].zip' log created by [AutoLogger](https://tools.safezone.cc/drongo/test/AutoLogger-test.zip)
 
**Please, note that only members of [VIRUSNET-Association](https://github.com/VIRUSNET-Association) are allowed to respond to PC cure topics. Ignore any recommendations given by other users, including PM !!!**

**Assistance is provided free of charge in our free time. If you found our help useful, you can thank us with any amount using [this form](https://safezone.cc/donate/) or you can leave feedback in [Guestbook](https://github.com/dragokas/hijackthis/issues/47).**

**Comment by coldcore321** [comment]
the problem is that on my computer (the one with the problem i came here with),  ms defender won't let  autologger open and the malware I have has hickjacked service entries so i can'/t disable defender AT ALL and it's blocking autologger. 
On thjis computer it tells me to get and updated version of auto logger and then aborts the program. So what do I do? I cant install the program and run it because of the malware I have . I'm at the end of my sanity with this 4 month
long mess that just gets worse and worse-I know I just came here to this forum but i've been battleing this sice december. I dont know what to do.

**Comment by Sandor-Helper** [comment]
Sorry, I gave you wrong link. [This one is correct](https://safezone.cc/resources/autologger-regist-drongo.59/download).

**Comment by coldcore321** [comment]
I still have windows defender blocking any autologging. I even cut it at group policy level and it didn't matter. That was before whatever it is emptied the zip file of the dl from the link . What do I do now?

**Comment by Sandor-Helper** [comment]
> still have windows defender blocking any autologging

Please show us the screenshot of this blocking.

**Comment by coldcore321** [comment]
Hi it was actually Windows Defender Smart Screen that was blocking Autologger and then just a few minutes ago I found this article that solved the issue with that: https://www.windowscentral.com/how-fix-app-has-been-blocked-your-protection-windows-10. Worked and am looking for the logs right now. Here it is:

[report1.log](https://github.com/dragokas/hijackthis/files/6293214/report1.log)

Is that correct? I mean did I do it right?
Thank you

**Comment by Sandor-Helper** [comment]
No you don't.
We need packed file named **CollectionLog_date_time.zip** situated inside Autologger folder.

**Comment by coldcore321** [comment]
Sorry I submitted the file without looking at it first. Okay I was finally able to run the program correctly without being blocked by malware. 
[CollectionLog-2021.04.12-06.55.zip](https://github.com/dragokas/hijackthis/files/6297264/CollectionLog-2021.04.12-06.55.zip)
Thank you

**Comment by Sandor-Helper** [comment]
Please update the safe database. See the [instruction here](https://github.com/dragokas/hijackthis/wiki/AVZ:-How-to-update-safe-files%3F).

Next: uninstall **Avast Driver Updater** as unwanted app.
Did you install **Cloudflare WARP** by yourself? If not, uninstall it also.

Then:
Download [AdwCleaner (by Malwarebytes)](https://toolslib.net/downloads/viewdownload/1-adwcleaner/) and save it to **Desktop**.
Run (it should be run by right-clicking **as Administrator**), press **"Scan"** and wait.
At the end of the scan log will be found at: 
**C:\AdwCleaner\Logs\AdwCleaner[Sxx].txt** (where **x** is any digit).
Attach it to your next post here.

**Comment by coldcore321** [comment]
Autologger didn't even scan the C drive. I don't know if it completed the scan properly at all 
and I can't upload log files on the referred page, I got this error message:    
 ![screenshot](https://user-images.githubusercontent.com/82003019/114791125-ca24b780-9d3a-11eb-9ec7-e14f376b52c8.png)![sreen2](https://user-images.githubusercontent.com/82003019/114791003-8df15700-9d3a-11eb-9c96-fdb9adc1e5de.png)

Log too big to go straight onto this page. Here's is a screen shot of AVZ when the scan was through. :

![screenshot](https://user-images.githubusercontent.com/82003019/114791125-ca24b780-9d3a-11eb-9ec7-e14f376b52c8.png)

Awcleaner: 
[0000000AdwCleaner[S06].txt](https://github.com/dragokas/hijackthis/files/6314167/0000000AdwCleaner.S06.txt)

What do I do now? Thank you

**Comment by coldcore321** [comment]
First image above is a duplicate of image underneath it.

**Comment by Sandor-Helper** [comment]
> Autologger didn't even scan the C drive

You don't have to check any disks in this window. AVZ scan all needed places as it should to.



> Log too big to go straight onto this page

Upload it to any cloud file exchange and give as a link to download it.

Next:
Please download [Farbar Recovery Scan Tool](https://www.bleepingcomputer.com/download/farbar-recovery-scan-tool/) and save it to your Desktop.

**Note**: You need to run the version compatible with your system. If you are not sure which version applies to your system download both of them and try to run them. Only one of them will run on your system, that will be the right version.

- Right click to run as administrator. When the tool opens click **Yes** to disclaimer.
- Press **Scan** button.
- It will produce logs called **FRST.txt** and **Addition.txt** in the same directory the tool is run from.
- Please attach the logs back here.

**Comment by coldcore321** [comment]
I had to reset windows 10 this morning , waking up to a keyboard that would only type nonsense. Running the Autologger again but here is what I have right now.  I'll post the other momentarily before I get blocked or messed with by this malware. 

[FRST.txt](https://github.com/dragokas/hijackthis/files/6332201/FRST.txt)
[Addition.txt](https://github.com/dragokas/hijackthis/files/6332202/Addition.txt)

**Comment by coldcore321** [comment]
and now i'm being blocked from uploading the autologger file on the referred page, it gets removed immediately. I had so many reote connections on my computer last night that this seems insurmountable. I have a back up but it's probably tainted too. Gotta do a wipe of everything? I 'll keep trying another autolgger log.

**Comment by coldcore321** [comment]
Okay it uploaded, so i'm assuming it gets to you guys somehow because it doesn't spit out a file to submit. That page last week I thought was an error page was a successful upload page, just in russian-okay understood.

**Comment by Sandor-Helper** [comment]
> a keyboard that would only type nonsense

What do you mean by this? Please describe it more detailed.
I still didn't find any maliciouse traces in any logs.
